### PR TITLE
Sync with Che master

### PR DESCRIPTION
**Pull request by garagatyi**
@vparfonov @ashumilova Please take a look

### CODE REVIEW

File: assembly/assembly-ide-war/pom.xml [hunk 1]
```diff
@@ -231,6 +231,10 @@
         </dependency>
         <dependency>
             <groupId>org.eclipse.che.plugin</groupId>
+            <artifactId>che-plugin-gwt-ext-gwt</artifactId>
```
Comment (vparfonov) [inline assembly/assembly-ide-war/pom.xml]: not need in Artik IDE

File: assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml [hunk 2]
```diff
@@ -48,6 +48,7 @@
     <inherits name='org.eclipse.che.ide.ext.java.jdi.JavaDebuggerExtension'/>
     <inherits name='org.eclipse.che.ide.extension.machine.Machine'/>
     <inherits name='org.eclipse.che.plugin.maven.Maven'/>
+    <inherits name='org.eclipse.che.ide.ext.gwt.GWT'/>
```
Comment (vparfonov) [inline assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml]: not need in Artik IDE

**Comment by ashumilova** [comment]
+1

File: assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml (hunk 2 above)
Comment (garagatyi) [inline assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml]: deleted

File: assembly/assembly-ide-war/pom.xml (hunk 1 above)
Comment (garagatyi) [inline assembly/assembly-ide-war/pom.xml]: deleted

**Comment by garagatyi** [comment]
@vparfonov I fixed lines where you had complaint

**Comment by vparfonov** [comment]
+1

**Pull request merged**